scenarios/bin/traffic.py

```diff
@@ -20,6 +20,7 @@
 pkt_count = 0
 
 ue_addresses_expected = set()  # UE addresses that we expect to receive packets for/from
+downlink_teids_expected = set() # Downlink TEIDs that we expect to receive packets for
```

Comment:
we don't care when doing buffering. It is ok to count packets but we don't care if the TEIDs are 0 for some